script/build_all_plugins_app.sh

```diff
@@ -4,14 +4,15 @@
 # sure all first party plugins can be compiled together.
 
 # So that users can run this script from anywhere and it will work as expected.
-SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
 REPO_DIR="$(dirname "$SCRIPT_DIR")"
 
 source "$SCRIPT_DIR/common.sh"
 check_changed_packages > /dev/null
 
 cd $REPO_DIR/examples/all_plugins
 flutter clean > /dev/null
+(cd "$REPO_DIR" && pub global run flutter_plugin_tools gen-pubspec --exclude firebase_core,firebase_ml_vision)
```

Comment:
Exclude `firebase_core` because all the other firebase plugins depend on it. This caused a problem when compiling. 

Exclude `firebase_ml_vision` because it requires iOS 9.0+ which caused a problem when compiling with `google_maps_flutter`. I didn't want to change either plugin in this PR.

Reply:
sgtm. note that we'll be moving these plugins out of this repo soon.